alignak/objects/contact.py

```diff
@@ -72,15 +73,15 @@ class Contact(Item):
         'contactgroups':    ListProp(default=[], fill_brok=['full_status']),
         'host_notifications_enabled': BoolProp(default=True, fill_brok=['full_status']),
         'service_notifications_enabled': BoolProp(default=True, fill_brok=['full_status']),
-        'host_notification_period': StringProp(fill_brok=['full_status']),
-        'service_notification_period': StringProp(fill_brok=['full_status']),
+        'host_notification_period': StringProp(default='', fill_brok=['full_status']),
+        'service_notification_period': StringProp(default='', fill_brok=['full_status']),
         'host_notification_options': ListProp(default=[''], fill_brok=['full_status'],
                                               split_on_coma=True),
         'service_notification_options': ListProp(default=[''], fill_brok=['full_status'],
                                                  split_on_coma=True),
         # To be consistent with notificationway object attributes
-        'host_notification_commands': ListProp(fill_brok=['full_status']),
-        'service_notification_commands': ListProp(fill_brok=['full_status']),
+        'host_notification_commands': ListProp(fill_brok=['full_status'], default=[]),
```

Comment:
To have same arguments order like other, define first _default_ and after _fill_brok_